atd-toolbox/data_model/python/90_fatalities_view.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from utils import save_file, make_migration_dir, save_empty_down_migration
+
+
+def load_sql_template(name):
+    with open(name, "r") as fin:
+        return fin.read()
+
+
+def main():
+    indexes_sql = load_sql_template("sql_templates/fatalities_view.sql")
+    migration_path = make_migration_dir("indexes")
```

Comment:
looks great! would you rename this from `indexes` to `fatalities_view` ?

Reply:
oh yeah,  fixed this!